nucliadb_vectors/src/service/reader.rs

```diff
@@ -93,6 +93,17 @@ impl VectorReader for VectorReaderService {
             formula.extend(clause_labels);
         }
 
+        if !request.field_filters.is_empty() {
+            for field in request.field_filters.iter() {
+                // split "a/title" into "a" and "title"
+                let parts = field.split('/').collect::<Vec<&str>>();
+                let field_type = parts.first().unwrap_or(&"").to_string();
+                let field_name = parts.get(1).unwrap_or(&"").to_string();
+                let clause = AtomClause::key_field(field_type, field_name);
+                formula.extend(clause);
```

Comment:
Careful with this, this makes an `AND` for all field types. So if multiple fields are specified, it won't match anything at all. You'll need to create a `CompoundClause::new(BooleanOperator::Or, field_clauses)` and add that to the formula instead.

Reply:
Maybe add a test as well?

Reply:
oh good catch! totally, I will add a test for this

Reply:
fixed!